Sources/Security/Signing.swift

```diff
@@ -312,7 +317,10 @@ private extension Signing {
     }
 
     /// - Returns: `nil` if the key is expired or has an invalid expiration date.
-    private static func extractAndVerifyIntermediateKeyExpiration(_ expirationData: Data) -> Date? {
+    private static func extractAndVerifyIntermediateKeyExpiration(
+        _ expirationData: Data,
+        _ clock: ClockType
```

Comment:
Hmm seems `clock` is unused? I'm guessing we want to use it to get the current date in line 332?

Reply:
good catch

Reply:
🤦‍♂️ 
This would have started failing in like 10 days 😅